addons/mail/static/tests/new/discuss/channel_member_list_tests.js

```diff
@@ -0,0 +1,165 @@
+/** @odoo-module **/
+
+import { start, startServer } from "@mail/../tests/helpers/test_utils";
+import { getFixture } from "@web/../tests/helpers/utils";
+
+let target;
+
+QUnit.module("mail", (hooks) => {
+    hooks.beforeEach(async () => {
+        target = getFixture();
+    });
+
+    QUnit.module("channel member list");
+
+    QUnit.test(
+        "there should be a button to show member list in the thread view topbar initially",
+        async function (assert) {
+            assert.expect(1);
```

Comment:
I would remove all `assert.expect()` counts in all tests altogether.
It was guidelines to add them back then, but now it's not mandatory.
It's slightly noisy to have them, hence why I prefer not having them in the end.

It might seem relevant in some cases, like when you have an assertion in a callback and uses this `assert.expect()` counter as a way to track amount of assert, so that it fails if callback is not called.
However, a much better approach now is to use `assert.step()` and `assert.verifySteps()`, which both of them do not need `assert.expect()` at all.

All this to say counting amount of assertion is useless, and just put more strain on code readability and test maintainability. So just remove it.